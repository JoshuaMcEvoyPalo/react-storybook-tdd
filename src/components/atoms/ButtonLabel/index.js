import React from 'react'
import './ButtonLabel.css'

const ButtonLabel = ({ text }) => {
  return (
    <div data-testid="button-label" className="button-label">{text}</div>
  )
}

export default ButtonLabel
