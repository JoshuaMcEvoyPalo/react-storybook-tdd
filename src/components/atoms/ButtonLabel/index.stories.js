import ButtonLabel from '.'

export default {
  title: 'Atoms / Button Label',
  component: ButtonLabel,
}

const Template = (args) => <ButtonLabel {...args} />

export const Default = Template.bind({})
Default.args = {
  text: "Sign in",
}