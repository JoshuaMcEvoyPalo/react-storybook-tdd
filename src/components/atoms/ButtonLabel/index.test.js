import renderer from 'react-test-renderer';
import ButtonLabel from '.'

describe('ButtonLabel tests', () => {
  it('Should render ButtonLabel correctly', () => {
    const tree = renderer.create(<ButtonLabel text="Sign in" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})