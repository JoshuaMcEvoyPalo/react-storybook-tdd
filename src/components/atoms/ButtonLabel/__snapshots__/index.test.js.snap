// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLabel tests Should render ButtonLabel correctly 1`] = `
<div
  className="button-label"
  data-testid="button-label"
>
  Sign in
</div>
`;
